src/lib/graphql/fetchAllProcessingActivities.ts

```diff
@@ -0,0 +1,133 @@
+import { GraphQLClient } from 'graphql-request';
+import { PROCESSING_ACTIVITIES } from './gqls';
+import { makeGraphQLRequest } from './makeGraphQLRequest';
+import type {
+  ProcessingPurpose,
+  DataCategoryType,
+} from '@transcend-io/privacy-types';
+import type { Region } from './formatRegions';
+
+export interface ProcessingActivity {
+  /** ID of processing activity */
+  id: string;
+  /** Title of processing activity */
+  title: string;
+  /** Description of processing activity */
+  description: string;
+  /** Security measure details */
+  securityMeasureDetails?: string;
+  /** Controllerships */
+  controllerships: string[];
+  /** Storage regions */
+  storageRegions: Region[];
+  /** Transfer regions */
+  transferRegions: Region[];
+  /** Retention type */
+  retentionType: string;
+  /** Retention period in days */
+  retentionPeriod?: number;
+  /** Data protection impact assessment link */
+  dataProtectionImpactAssessmentLink?: string;
+  /** Data protection impact assessment status */
+  dataProtectionImpactAssessmentStatus: string;
+  /** Attribute values */
+  attributeValues: {
+    /** Name of attribute value */
+    name: string;
+    /** Attribute key */
+    attributeKey: {
+      /** Name of attribute key */
+      name: string;
+    };
+  }[];
+  /** Data silos */
+  dataSilos: {
+    /** Data silo ID */
+    id: string;
+    /** Data silo title */
+    title: string;
+  }[];
+  /** Data subjects */
+  dataSubjects: {
+    /** Data subject ID */
+    id: string;
+    /** Data subject type */
+    type: string;
+  }[];
+  /** Teams */
+  teams: {
+    /** Team ID */
+    id: string;
+    /** Team name */
+    name: string;
+  }[];
+  /** Owners */
+  owners: {
+    /** Owner ID */
+    id: string;
+    /** Owner email */
+    email: string;
+  }[];
```

Comment:
Looking at teams and owners on other `fetchAll***` functions, it looks like we generally don't get the ids, for any of these related resources (dataSubjects, processingPurposeSubCategories, etc.).